src/flows/index.ts

```diff
@@ -27,8 +27,8 @@ const context = require.context(
   true,
   /^\.\/([^\/])+\/index\.(ts|tsx)$/
 )
-context.keys().forEach(key => {
-  const module = context(key)
+context.keys().forEach(async key => {
+  const module = await context(key)
   module.default((definition: TypeRegistration) => {
```

Comment:
What was the need for this change driven by?

Reply:
@wdoconnell there's no documentation around this, but it looks like webpack 5 changed how [`require.context`](https://webpack.js.org/guides/dependency-management/) works so that it's asynchronous.

More snarkily: the change was driven by the app not running and pointing to these lines with an error that said something to the effect of `default` doesn't exist on a promise.

Reply:
👍 🚀